docling_core/types/doc/document.py

```diff
@@ -3885,6 +3885,7 @@ def export_to_doctags(  # noqa: C901
         add_table_cell_location: bool = False,
         add_table_cell_text: bool = True,
         minified: bool = False,
+        pages: Optional[list[int]] = None,
```

Comment:
Let's make it a set since the underlying parameter is also a set.
```suggestion
        pages: Optional[set[int]] = None,
```